src/styles/variables.scss

```diff
@@ -6,7 +6,7 @@ $darkBlue: #001e49;
 $grey: #a4adb6;
 $lightBlue: #325e91;
 $lightCyan: #4dc7bc;
-$lightGrey: #e5eaef;
+$lightGrey: #020202;
```

Comment:
Die Farben sind in den Visuals fest definiert. Bitte keine Abweichungen. Woher kommt diese Anpassung?